src/metatensor/models/soap_bpnn/tests/test_invariance.py

```diff
@@ -12,8 +13,17 @@
 def test_rotational_invariance():
     """Tests that the model is rotationally invariant."""
 
-    all_species = [1, 6, 7, 8]
-    soap_bpnn = Model(all_species, DEFAULT_HYPERS["model"]).to(torch.float64)
+    capabilities = ModelCapabilities(
+        length_unit="Angstrom",
+        species=[1, 6, 7, 8],
+        outputs={
+            "energy": ModelOutput(
+                quantity="energy",
+                unit="eV",
```

Comment:
My reasoning is that all models need to be exportable, and they need that capabilities object when they are exported. I thought that we might as well make things clear from the beginning and use it at initialization rather than trying to recover it when the model is exported. I also find that these classes fit very well with our future projects with this library

Reply:
Lowercase or uppercase does not matter here, MetatensorAtomisticModel will normalize before doing any comparison.